crates/factor-outbound-networking/src/lib.rs

```diff
@@ -65,15 +69,19 @@ impl Factor for OutboundNetworkingFactor {
             })
             .collect::<anyhow::Result<_>>()?;
 
-        let runtime_config = match ctx.take_runtime_config() {
-            Some(cfg) => cfg,
-            // The default RuntimeConfig provides default TLS client configs
-            None => RuntimeConfig::new([])?,
-        };
+        let RuntimeConfig {
+            client_tls_configs,
+            blocked_ip_networks: block_networks,
+            block_private_networks,
+        } = ctx.take_runtime_config().unwrap_or_default();
```

Comment:
As mentioned below, I'm unsure of the use of `unwrap_or_default` here is correct when previously it was defaulting to `RuntimeConfig::new([])`.